test/integration/helpers/inspec/spec_helper.rb

```diff
@@ -0,0 +1,36 @@
+# -*- encoding : utf-8 -*-
+
+require 'net/http'
+require 'open-uri'
+require 'uri'
```

Comment:
These requires need to be moved into ``test/cookbooks/jenkins_test/files/default/jenkin_is_ready.rb``. In addition, you need to add:
``` ruby
require 'openssl'
```

Which was likely pulled into ServerSpec by default before.

Reply:
Resolved with 2f2bee8